lib/recurly/pager.php

```diff
@@ -45,15 +44,14 @@ public function rewind() {
    */
   public function current()
   {
-    if (empty($this->_count)) {
+    if ($this->count() == 0) {
```

Comment:
yeah that was some setup for when i thought i was going to extract the per page part of the enumeration. so yeah no longer needed.